spec/queries/affected_by_recurring_blackout_query.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe AffectedByRecurringBlackoutQuery do
+  let!(:overdue) do
+    create(:overdue_reservation, due_date: Time.zone.today - 1.day)
+  end
+  let!(:checked_out) do
+    create(:checked_out_reservation, due_date: Time.zone.today + 1.day)
+  end
+  let!(:no_problem) do
+    create(:checked_out_reservation, due_date: Time.zone.today + 4.days)
+  end
+  let!(:archived) { create(:archived_reservation) }
+  let!(:checked_in) { create(:checked_in_reservation) }
+  let!(:reserved) do
+    create(:valid_reservation,
+           start_date: Time.zone.today + 1.day,
+           due_date: Time.zone.today + 2.days)
+  end
+  let!(:res_dates) do
+    [Time.zone.today.next, Time.zone.today.next + 1.day]
```

Comment:
I think there's a `Time.zone.tomorrow` method that might be a little more concise here.